### PR TITLE
Azure put get support

### PR DESCRIPTION
**Pull request by smtakeda**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -12,6 +12,7 @@ enable_testing()
 add_definitions(-DLOG_USE_COLOR)
 
 option(BUILD_TESTS "True if build tests" on)
+set(OPENSSL_VERSION_NUMBER  0x11100000L)
```
Comment (notkriswagner) [inline CMakeLists.txt]: Will this cause the build to fail if the version of OpenSSL that a customer is building with doesn't match this version of OpenSSL?

File: scripts/build_azuresdk.sh [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    azure_configure_opts+=("-DCMAKE_BUILD_TYPE=Release")
+fi
+azure_configure_opts+=(
+    "-DCMAKE_VERBOSE_MAKEFILE:BOOL=OFF"
+    "-DCMAKE_C_COMPILER=$GCC"
+    "-DCMAKE_CXX_COMPILER=$GXX"
+    "-DCMAKE_INSTALL_PREFIX=$AZURE_BUILD_DIR"
+    "-DBUILD_SHARED_LIBS=OFF"
+    "-DUSE_OPENSSL=true"
+    "-DOPENSSL_VERSION_NUMBER=0x11100000L"
```
Comment (smtakeda) [inline scripts/build_azuresdk.sh]: @rvishureddy , Better moving this version to the top of this file or config. Related to @notkriswagner question; does this matter to Azure SDK build? I mean without the version number, what will happen? Ideally version neutral build scrip would be great for maintenance.

Comment (rvishureddy) [reply scripts/build_azuresdk.sh]: Openssl API's changed with version 1.1.1. libSnowflakeclient and ODBC both use openssl 1.1.1b and to have corresponding API's invoked in Azure sdk it should be compiled with the above flag.

Comment (smtakeda) [reply scripts/build_azuresdk.sh]: Ok, thanks for confirmation.

**Comment by smtakeda** [comment]
LGTM

File: CMakeLists.txt (hunk 1 above)
Comment (rvishureddy) [reply CMakeLists.txt]: They should not fail. As we statically link and do archive all libs into libSnowflake.so the application should link with the client specific openssl.

File: lib/client.c [hunk 3]
```diff
@@ -456,6 +456,7 @@ _snowflake_check_connection_parameters(SF_CONNECT *sf) {
     log_debug("timezone: %s", sf->timezone);
     log_debug("login_timeout: %d", sf->login_timeout);
     log_debug("network_timeout: %d", sf->network_timeout);
+    log_debug("snowflake_test_ca_bundle_file from getenv: %s", getenv("SNOWFLAKE_TEST_CA_BUNDLE_FILE"));
```
Comment (notkriswagner) [inline lib/client.c]: This looks like a test debugging line, could this be moved out of client.c and into one of the test files maybe?

**Comment by notkriswagner** [comment]
Besides minor issue, LGTM.

**Comment by smtakeda** [comment]
ok, let's merge the change.

**Pull request merged**